hr_attendance_geolocation/views/hr_attendance_views.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="hr_attendance_view_form" model="ir.ui.view">
```

Comment:
The form is not opened on 14.0, as it is an editable tree. Can you check it please

Reply:
You can switch to kanban view and click on the kanban card to check the form details.

Reply:
Great! You're right but it would be interesting if it could be opened on 14.0